app/jobs/export/backup_mitglieder_export_job.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2024, Schweizer Alpen-Club. This file is part of
+#  hitobito_sac_cas and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sac_cas
+
+class Export::BackupMitgliederExportJob < RecurringJob
+
+  run_every 1.day
+  self.use_background_job_logging = true
+
+  ROLE_TYPES_TO_BACKUP = [::Group::Sektion, ::Group::Ortsgruppe].freeze
+
+  def initialize
+    super
+    @errors = []
+  end
+
+  def perform_internal
+    relevant_groups.find_each do |group|
```

Comment:
Ich weis nicht wie lange so ein Export geht aber wenn wir das in einem Job machen blockieren wir einen Worker mitunter sehr lange. Ich fände es besser, wenn wir einen job pro Gruppe enqueuen würden. Eventuell einfach `BackupMitgliederExport` zu einem `BackupMitgliederExportJob` und diesen zu einem `BackupMitgliederScheduleJob` machen